lib/pysquared/logger.py

```diff
@@ -50,10 +78,18 @@ def _log(self, level: str, level_value: int, message: str, **kwargs) -> None:
         ):
             kwargs["err"] = traceback.format_exception(kwargs["err"])
 
-        json_output = json.dumps(kwargs)
+        json_order: OrderedDict[str, str] = OrderedDict(
+            [("time", asctime), ("level", level), ("msg", message)]
+        )
+        json_order.update(kwargs)
+
+        json_output = json.dumps(json_order)
 
         if self._can_print_this_level(level_value):
-            print(json_output)
+            colored_json_output = json_output.replace(
+                f'"level": "{level}"', f'"level": "{LogColors[level]}"'
+            )
+            print(colored_json_output if self.colorized else json_output)
```

Comment:
I find myself looking at this section and contemplating if it would better be refactored as something like:
```
if self._can_print_this_level(level_value):
    if self.colorized:
        json_output = json_output.replace(f'"level": "{level}"', f'"level": "{LogColors[level]}"')
    print(json_output)
```
The way the current code is, even if self.colorized is False, the code would still make an object above with color, which I think is somewhat unnecessary, as it would go on to completely disregard it and use the plain json output object. 

The proposed code idea shared here would result in the json object with color being created only when self.colorized is True.

@nateinaction @Mikefly123 Do you have any thoughts on this suggestion from me? I know that we aren't in chase of the most extreme performance but preventing unnecessary object creation doesn't hurt

Reply:
Sounds like a good optimization, go ahead and commit it!
```suggestion
            if self.colorized:
                json_output = json_output.replace(f'"level": "{level}"', f'"level": "{LogColors[level]}"')
            print(json_output)
```